spring-pulsar-reactive/src/main/java/org/springframework/pulsar/reactive/core/ReactivePulsarTemplate.java

```diff
@@ -34,7 +34,7 @@
 import reactor.core.publisher.Mono;
 
 /**
- * A thread-safe template for executing high-level reactive Pulsar operations.
+ * A template for executing high-level reactive Pulsar operations.
```

Comment:
why did we remove the thread-safe part from the comment?

Reply:
I removed it because it was a holdover from the Kafka template and we don't really distinguish this elsewhere. 

If we want to bring it back, I would propose that we use @ThreadSafe and also do it for all classes that fit that bill rather than just the template.